plugin-server/functional_tests/exports-v2.test.ts

```diff
@@ -75,69 +94,150 @@ test.concurrent(`exports: historical exports v2`, async () => {
     // First let's capture an event and wait for it to be ingested so
     // so we can check that the historical event is the same as the one
     // passed to processEvent on initial ingestion.
-    await capture(producer, teamId, distinctId, uuid, '$autocapture', {
+    const eventTime = new Date('2022-01-01T00:00:00.000Z')
+    const sentAt = new Date('2022-01-02T00:00:00.000Z')
+    const now = new Date('2022-01-03T00:00:00.000Z')
+    const skewAdjustedTimestamp = new Date(now.getTime() - (sentAt.getTime() - eventTime.getTime()))
```

Comment:
nit: with these values, we'd have `skewAdjustedTimestamp === sentAt` so we could miss a regression. WDYT about explictly specifying `skewAdjustedTimestamp` and have different deltas (closer to the typical use case, client is ten minutes out of sync) to avoid values overlapping?
```suggestion
    const eventTime = new Date('2022-01-01T05:08:00.000Z')
    const sentAt = new Date('2022-01-01T05:10:00.000Z')
    const now = new Date('2022-01-01T05:00:00.000Z')
    const skewAdjustedTimestamp = new Date('2022-01-01T04:58:00.000Z')
```

Didn't try to run the tests with these values though, test before committing